test/Microsoft.AspNetCore.Mvc.IntegrationTests/MutableObjectModelBinderIntegrationTest.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace Microsoft.AspNetCore.Mvc.IntegrationTests
 {
-    // Integration tests targeting the behavior of the MutableObjectModelBinder and related classes
+    // Integration tests targeting the behavior of the ComplexTypeModelBinder and related classes
     // with other model binders.
     public class MutableObjectModelBinderIntegrationTest
```

Comment:
(I should i reverted this one)I wanted to avoid the renaming this class and files etc. as it could mask the intended changes